website/src/pages/admin/edit/[id].tsx

```diff
@@ -0,0 +1,176 @@
+  const labelColor = useColorModeValue("gray.600", "gray.400");
+  const cardColor = useColorModeValue("gray.50", "gray.800");
+
+  return (
+    <>
+      <Head>
+        <title>{t("common:title")}</title>
+        <meta name="description" content={t("common:conversational")} />
+      </Head>
+      {(isLoading || !data) && (
+        <Card>
+          <CardBody>
+            <MessageLoading />
+          </CardBody>
+        </Card>
+      )}
+      {error && "Unable to load message tree"}
+      {!isLoading && data && (
+        <AdminArea>
+          <TwoColumnsWithCards>
+            <>
+              <Stack spacing={1}>
+                <Text fontSize="xl" fontWeight="bold" color={titleColor}>
+                  {t("common:edit")}
+                </Text>
+                <Text fontSize="md" color={labelColor}>
+                  {t("tasks:moderator_edit_explain")}
+                </Text>
+              </Stack>
+              <Box marginTop={4} borderRadius={"lg"} backgroundColor={cardColor} className="p-3 sm:p-6">
+                {!messageThread ? "Unable to build tree" : <MessageTree tree={messageThread} messageId={id} />}
+              </Box>
+            </>
+            <>
+              <Stack spacing={1}>
+                <Tabs isLazy>
+                  <TabList>
+                    <Tab>{t("tasks:tab_write")}</Tab>
+                    <Tab>{t("tasks:tab_preview")}</Tab>
+                  </TabList>
+                  <TabPanels>
+                    <TabPanel>
+                      <TrackedTextarea
+                        text={messageText}
+                        onTextChange={setMessageText}
+                        thresholds={{ low: 20, medium: 40, goal: 50 }}
+                        textareaProps={{ minRows: 5 }}
+                      />
+                    </TabPanel>
+                    <TabPanel p="0" pt="4" marginBottom={2}>
+                      {previewContent}
+                    </TabPanel>
+                  </TabPanels>
+                </Tabs>
+                <Flex justify={"flex-start"}>
+                  <Button
+                    size="lg"
+                    variant="solid"
+                    colorScheme="green"
+                    onClick={handleSubmit}
+                    isLoading={submitting}
+                    loadingText={"Submitting"}
+                  >
+                    {t("common:submit")}
+                  </Button>
+                </Flex>
+              </Stack>
+            </>
+          </TwoColumnsWithCards>
+        </AdminArea>
+      )}
+    </>
+  );
+};
+
+EditMessage.getLayout = AdminLayout;
+
+export const getServerSideProps: GetServerSideProps<{ id: string }, { id: string }> = async ({
+  locale = "en",
+  params,
+}) => ({
+  props: {
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    id: params!.id,
+    ...(await serverSideTranslations(locale, ["common", "message", "labelling", "tasks"])),
```

Comment:
no need to define the individual namespaces here, `await serverSideTranslations(locale)` is good enough.